soe/src/contrib.make

```diff
@@ -133,7 +133,7 @@ projects[globalredirect][version] = "1.5"
 projects[google_analytics][subdir] = "contrib"
 projects[google_analytics][version] = "2.6"
 projects[google_tag][subdir] = "contrib"
-projects[google_tag][version] = "1.2"
+projects[google_tag][version] = "1.6"
```

Comment:
No upgrade to 2.1?

Reply:
https://www.drupal.org/project/google_tag/releases/7.x-2.1

Reply:
I went conservative.